plugins/modules/network/routeros/routeros_api.py

```diff
@@ -0,0 +1,491 @@
+notes:
+  - I(add), I(remove), I(update), I(cmd) and I(query) are mutually exclusive.
+  - I(check_mode) is not supported
+requirements:
+  - librouteros
+  - Python >= 3.6 (for librouteros)
+options:
+  hostname:
+    description:
+      - RouterOS hostname API.
+    required: true
+    type: str
+  username:
+    description:
+      - RouterOS login user.
+    required: true
+    type: str
+  password:
+    description:
+      - RouterOS user password.
+    required: true
+    type: str
+  ssl:
+    description:
+      - If is set TLS will be used for RouterOS API connection.
+    required: false
+    type: bool
+  port:
+    description:
+      - RouterOS api port. If ssl is set, port will apply to ssl connection.
+        Defaults are C(8728) for the HTTP API,  and C(8729) for the HTTPS API.
+    type: int
+  path:
+    description:
+      - Main path for all other arguments.
+        If other arguments are not set, api will return all items in selected path.
+        Example C(ip address). Eqvivalent of RouterOS CLI C(/ip address print).
+    required: true
+    type: str
+  add:
+    description:
+      - Will add selected arguments in selected path to RouterOS config.
+        Example C(address=1.1.1.1/32 interface=ether1).
+        Equivalent in RouterOS CLI C(/ip address add address=1.1.1.1/32 interface=ether1).
+    type: str
+  remove:
+    description:
+      - Remove config/value from RouterOS by '.id'.
+         Example C(*03) will remove config/value with C(id=*03) in selected path.
+          Equivalent in RouterOS CLI C(/ip address remove numbers=1).
+          Note C(number) in RouterOS CLI is different from C(.id).
+    type: str
+  update:
+    description:
+      - Update config/value in RouterOS by '.id' in selected path.
+         Example C(.id=*03 address=1.1.1.3/32) and path C(ip address)
+         will replace existing ip address with C(.id=*03).
+         Equivalent in RouterOS CLI
+         C(/ip address set address=1.1.1.3/32 numbers=1).
+         Note C(number) in RouterOS CLI is different from C(.id).
+    type: str
+  query:
+    description:
+      - Query given path for selected query attributes from
+         RouterOS aip and return '.id'.
+          WHERE is key word which extend query. WHERE format is
+         key operator value - with spaces.
+          WHERE valid operators are C(==), C(!=), C(>), C(<).
+          Example path C(ip address) and query C(.id address) will return
+         only C(.id) and C(address) for all items in C(ip address) path.
+          Example path C(ip address) and
+          query C(.id address WHERE address == 1.1.1.3/32)
+         will return only C(.id) and C(address) for items in C(ip address)
+          path, where address is eq to 1.1.1.3/32.
+          Example path C(interface) and query C(mtu name WHERE mut > 1400) will
+         return only interfaces C(mtu,name) where mtu is bigger than 1400.
+          Equivalent in RouterOS CLI C(/interface print where mtu > 1400).
```

Comment:
```suggestion
      - Query given path for selected query attributes from RouterOS aip and return '.id'.
      - WHERE is key word which extend query. WHERE format is key operator value - with spaces.
      - WHERE valid operators are C(==), C(!=), C(>), C(<).
      - Example path C(ip address) and query C(.id address) will return only C(.id) and C(address) for all items in C(ip address) path.
      - Example path C(ip address) and query C(.id address WHERE address == 1.1.1.3/32)
        will return only C(.id) and C(address) for items in C(ip address) path, where address is eq to 1.1.1.3/32.
      - Example path C(interface) and query C(mtu name WHERE mut > 1400) will
        return only interfaces C(mtu,name) where mtu is bigger than 1400.
        Equivalent in RouterOS CLI C(/interface print where mtu > 1400).
```
1. would be good to see these lines aligned and it's fair for the other params as well.
2. and `WHERE` is a keyword, right? Repetitions look a bit awkward. maybe to reformulate that somehow? It's just a suggestion.